# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/dietlibc/pkg_patch/pkg_bin86.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- bin86-0.16.12/ld/Makefile.orig	Wed Mar 20 21:58:20 2002
+++ bin86-0.16.12/ld/Makefile	Sun Sep  7 22:43:43 2003
@@ -17,7 +17,7 @@
 
 # An alternative file for a non-standard a.out.h (eg i386 linux on an Alpha)
 #
-# NATIVE=-DA_OUT_INCL='"a_out_local.h"' 
+NATIVE=-DA_OUT_INCL='"x86_aout.h"' 
 
 OBJS= dumps.o io.o ld.o readobj.o table.o typeconv.o linksyms.o \
       writex86.o writebin.o writerel.o
